Makefile

```diff
@@ -76,6 +76,13 @@ clean:
 test:
 	go test -v -race ./cmd/... ./pkg/...
 
+.PHONY: test/coverage
+test/coverage:
+	go test -v -race -coverprofile=cover.out ./cmd/... ./pkg/...
```

Comment:
np: could probably ditch the `-v` and `-race` as this is separate from `make test`
/lgtm

Reply:
Thanks - changed ✅